inference/core/workflows/core_steps/models/foundation/lmm_classifier.py

```diff
@@ -84,12 +89,17 @@ class BlockManifest(WorkflowBlockManifest):
         examples=["xxx-xxx", "$inputs.api_key"],
     )
 
+    @classmethod
+    def accepts_batch_input(cls) -> bool:
```

Comment:
[not a blocker] could this be inferred by type of parameters on run method of class?

Reply:
Depends I guess:
* the fact that step is SIMD (takes batch oriented data and applies the same processing) is detected and handled without this method being used
* for SIMD steps we use this method to see if block expects elements of the batch to be unpacked by EE or not - inferring that from signature may be a fragile code and my hunch is that we should not be doing it, as the standard practice for people may be to implement `run(...)` without type hints which are optional and in Python only small pool of tools (including for instance Pydantic) analyses type hints annotations as first order of business